src/System.Data.SqlClient/src/System/Data/SqlClient/LocalDBAPI.cs

```diff
@@ -101,6 +101,7 @@ private static LocalDBFormatMessageDelegate LocalDBFormatMessage
         {
             get
             {
+#if !MANAGED_SNI
```

Comment:
Was this change done for the issue or did you change this because its approproate to disable local db for managed SNI?